components/heap/include/heap_trace.inc

```diff
@@ -39,7 +39,7 @@ inline static uint32_t get_ccount(void)
 #define STACK_OFFSET  2
 
 #define TEST_STACK(N) do {                                              \
-        if (STACK_DEPTH == N) {                                         \
+        if (N >= STACK_DEPTH) {                                         \
             return;                                                     \
```

Comment:
I'm not really sure what this should be. Why was it originally STACK_DEPTH == N?